detox/android/detox/src/main/java/com/wix/detox/espresso/DetoxMatcher.java

```diff
@@ -75,6 +77,10 @@ private DetoxMatcher() {
         return isDisplayingAtLeast(75);
     }
 
+    public static Matcher<View> matcherForMinimumVisiblePercent(double percentage) {
+        return isDisplayingAtLeast((int)Math.floor(percentage * 100));
```

Comment:
I believe `Math.floor` isn't really needed here as the int-cast has the same effect.